Student/src/main/java/org/example/servlets/AddStudentServlet.java

```diff
@@ -0,0 +1,30 @@
+package org.example.servlets;
+
+import org.example.entities.Student;
+import org.example.model.Model;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AddStudentServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        getServletContext().getRequestDispatcher("/WEB-INF/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String surname = req.getParameter("surname");
+        Integer age = Integer.valueOf(req.getParameter("age"));
+        Student student = new Student(name, surname, age);
+        Model studentModel = Model.getStudentModel();
+        studentModel.add(student);
+
+        req.setAttribute("student", name);
+        doGet(req, resp);
```

Comment:
It's better to avoid calling methods such way (I mean if we need some logic from doGet method inside doPost, it' better either to copy paste the logic or move it to a separate private method and use this method both in doGet and doPost)